scalameta/semanticdb-scalac/src/main/scala/scala/meta/internal/semanticdb/PrinterOps.scala

```diff
@@ -10,8 +10,10 @@ import scala.reflect.internal.ModifierFlags._
 trait PrinterOps { self: DatabaseOps =>
   import g._
 
-  def showSynthetic(tpe: g.Type): AttributedSynthetic = {
-    showSynthetic(g.TypeTree(tpe))
+  def showSynthetic(tpe: g.Type): AttributedSynthetic = tpe match {
+    // Skip signature for abstract type members, e.g. type T <: Int
```

Comment:
Why's that? We can use Type.Bounds here, I think, wrapped in a Type.Lambda if necessary.

Reply:
I thought it would be best to skip these since I have no need for them at the moment, we can handle them later if they turn out to be useful.